layout/layout.go

```diff
@@ -733,11 +730,33 @@ func (i *Image) mutateCreatedAt(base v1.Image, created v1.Time) error {
 
 // mutateImage wraps the provided v1.Image into a layout.Image
 func (i *Image) mutateImage(base v1.Image) {
-	i.Image = &Image{
-		Image: base,
+	manifest, _ := base.Manifest()
```

Comment:
Is there a reason not to check the error?

Reply:
Done